.github/workflows/security.yaml

```diff
@@ -0,0 +1,25 @@
+name: Security vulnerabilities scan
```

Comment:
I have two doubs, 
 - why not using the official snyk action? https://github.com/snyk/actions
 - we are using the cli and not the webhook since we are planning to move to go modules?

Reply:
This action would fail from PR opened from forks:
```
Secrets are not passed to workflows that are triggered by a pull request from a fork.
```

Reply:
I could not make the official snyk action work because we are NOT using go modules and so it requires using a very specific path for the code. I'll see if I now can make it work

Reply:
It's not working. It's either impossible or even more hackish to make the official snyk action work with our code.
We need to execute a govendor sync (because we don't store the vendor folder) and also "hack" the GOPATH.
I think it's even uglier that using the makefile. When we move to go modules we can make it cleaner

Reply:
Yes I know it will fail from forks. We can remove the PR but then we will only test when we merge to master which may fail and so we get a "red" main branch. 
If we see that there are people using forks to contribute we may review this.